yt/yt/library/auth_server/cypress_cookie_login.cpp

```diff
@@ -78,8 +78,20 @@ class TCypressCookieLoginHandler
         const IRequestPtr& req,
         const IResponseWriterPtr& rsp)
     {
-        auto replyAndLogError = [&] (const TError& error, const std::optional<std::string>& user = {}) {
-            ReplyError(rsp, error);
+        auto replyAndLogError = [&] (
+            const TError& error,
+            const std::optional<std::string>& user = {},
+            bool replyWithGenericError = false)
```

Comment:
I'd prefer `bool maskError` or something like this, but current option is fine, so up to you.